canvas/services/event.py

```diff
@@ -44,8 +45,9 @@ def get_question_stats(question):
 
     answers = {}
     if isinstance(question, MultipleChoiceQuestion):
+        choices = json.loads(question.choices) if type(question.choices) == str else question.choices
```

Comment:
In what situation do you get `question.choices` to be str?

Reply:
I am unsure, I took this line from course/models/models.py get_rendered_choices (line 323 specifically) which the original code called instead. I presume it is in there for a reason so I didn't want to mess with it.